link-grammar/dict-common/dict-locale.c

```diff
@@ -441,6 +441,10 @@ bool dictionary_setup_defines(Dictionary dict)
 
 	dict->shuffle_linkages = false;
 
+	// Used for unattached quote marks, in the English dict only.
+	dict->zzz_connector = linkgrammar_get_dict_define(dict, EMPTY_CONNECTOR);
+	dict->zzz_connector = string_set_add(dict->zzz_connector, dict->string_set);
```

Comment:
To avoid a crash in case there is no definition:
``` c
if (NULL != dict->zzz_connector)
	dict->zzz_connector = string_set_add(dict->zzz_connector, dict->string_set);
```

Reply:
Dohh. Yes, of course.